Monika After Story/game/script-topics.rpy

```diff
@@ -13185,6 +13185,38 @@ label mas_show_unseen:
     m 3hua "There you go!"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_intrusive_thoughts",
+            category=['Psychology'],
```

Comment:
```suggestion
            category=['psychology'],
```
Not sure how we all missed this in 4 reviews, but categories must be in all lowercase letters. Using an uppercase will create an entirely new category.